.meta/sinks/datadog_logs.toml.erb

```diff
@@ -47,6 +47,7 @@ description = "The endpoint to stream logs to."
 <%= render(
   "_partials/fields/_tls_connector_options.toml",
   namespace: "sinks.datadog_logs.options",
+  enabled: true,
   can_enable: true,
```

Comment:
I don't think this is right. There is no `enabled` option for datadog_logs (that is only present in `TlsConfig`), so `can_enable` should be `false`, and the new `enabled` flag is irrelevant.

Reply:
So what I understand, we want to let the user enable it but the default option is to set `enabled = true`. So setting can_enable false doesn't work here. The `enabled` option is to set just the default.

Reply:
Without using `TlsConfig` in the code, there is no `enabled` option that may be used.